### PR TITLE
DROOLS-3092: [DMN Designer] User can not see all the definition editing custom data types

### PR DESCRIPTION
**Pull request by danielzhe**
DROOLS-3092: [DMN Designer] User can not see all the definition editing custom data types

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.java [hunk 1]
```diff
@@ -377,6 +377,12 @@ public void hideDataTypeNameInput() {
     @Override
     public void setupSelectComponent(final DataTypeSelect typeSelect) {
         type.innerHTML = "";
+
+        HTMLElement element = typeSelect.getElement();
```
Comment (karreiro) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.java]: I think we could add a final here:
```suggestion
        final HTMLElement element = typeSelect.getElement();
```

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.java [hunk 2]
```diff
@@ -377,6 +377,12 @@ public void hideDataTypeNameInput() {
...
+        if (element instanceof HTMLDivElement) {
```
Comment (karreiro) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.java]: We could remove this condition..

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/common/MenuInitializer.java [hunk 3]
```diff
@@ -29,18 +29,21 @@
...
-public class KebabMenuInitializer {
+public class MenuInitializer {
 
-    private final HTMLDivElement kebabMenu;
+    private final HTMLDivElement menu;
+    private final String dropDownClass;
 
-    public KebabMenuInitializer(final HTMLDivElement kebabMenu) {
-        this.kebabMenu = kebabMenu;
+    public MenuInitializer(final HTMLDivElement menu,
```
Comment (karreiro) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/common/MenuInitializer.java]: ..by changing this to:
```java
public MenuInitializer(final HTMLElement menu,
                       final String dropDownClass) {
// ...
```

**Comment by jomarko** [comment]
@danielezonca Hi, I will reviews once @karreiro suggestions incorporated.

**Comment by danielzhe** [comment]
Thank you for your review, @karreiro ! I did the  changes you suggested. Please, review when you have time. Thanks!

**Comment by karreiro** [comment]
Broken test:
```
org.kie.workbench.common.screens.datamodeller.backend.server.indexing.ImpactAnalysisJavaFileTest.testReferenceQueryInfrastructure
```

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by danielzhe** [comment]
@karreiro I have no idea about what is this broken test or what is his relation with this PR. I don't think is related.

**Comment by manstis** [comment]
Test failure is unrelated. https://github.com/kiegroup/appformer/pull/538 fixes it.

**Comment by manstis** [comment]
@danielzhe We need a _green_ build before merge... there has been a failing test in `kie-wb-common` recently but @ederign has kindly fixed.. so this should now build OK.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by ederign** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by ederign** [comment]
@manstis the build is green. :)

**Pull request merged**